webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require("path");
+const nodeExternals = require("webpack-node-externals");
+
+module.exports = {
+  entry: "./server.js",
+  target: "node",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "dist"),
+  },
+  resolve: {
+    modules: [path.resolve(__dirname, "node_modules")],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: [/node_modules/, /test/],
+      },
```

Comment:
Do we remove tests as well,
Do we not run the tests after the build is done?

How is this generally do, like do tests run and then build takes place or do build run then test runs?

Reply:
Since the test supporting dependencies were dev and we were removing them while bundling, so test wouldn;t work.